x/ccv/provider/client/cli/tx.go

```diff
@@ -223,9 +223,9 @@ Note that the one that signs this message is the owner of this consumer chain. T
 changed by updating the consumer chain.
 
 Example:
-%s tx provider create-consumer [chain-id] [path/to/metadata.json] [path/to/initialization-parameters.json] [path/to/power-shaping-parameters.json] --from node0 --home ../node0 --chain-id $CID
+%s tx provider create-consumer [path/to/create_consumer.json] --from node0 --home ../node0 --chain-id $CID
```

Comment:
Why do we need one json file as input?

Reply:
If you change, also change the Use description above.

Reply:
easier from a usability perspective IMO, as you have to deal with only one file.